examples/vhdl/docker_run_precompiled.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+cd $(dirname "$0")
+
+docker run --rm -t \
+  -v /$(pwd)/../..://work \
+  -w //work/examples/vhdl \
+  -e PYTHONPATH=//work \
+  ghdl/vunit:llvm-master sh -c ' \
+    VUNIT_SIMULATOR=ghdl; \
```

Comment:
Since the shell commands are half of the file, I'd recommend saving it to a different file. That will make it easier to read.
Note that the container is executed in the same place as the script. Hence, `... ghdl/vunit:llvm-master ./myscript.sh` should work.

Reply:
Good point

Reply:
@umarcor Anything else you want me to fix for this commit?